package.json

```diff
@@ -101,5 +101,6 @@
     "github": {
       "release": true
     }
-  }
+  },
+  "packageManager": "pnpm@8.1.1"
```

Comment:
Initially, I have gone with `package-manager-strict=false` in the `.npmrc` file vs this change. What are your thoughts?